### ISSUE TITLE
[ENHANCEMENT] - Changing the Techstack from Vanilla Javascript to React

### ISSUE THREAD

**Issue by VivekChatterjee** [VivekChatterjee/portfolio-site#1]
Need to Implement the web app using React which will eventually make the code shorter and clean.
Do needful changes for integrating React in the current repository.

**Comment by raviikushh** [comment]
- So you want the same output by using React.. right..? 
- I can make it. Assign me I'll start working on it now ownwards.

**Comment by VivekChatterjee** [comment]
Yes @raviikushh go ahead

**Comment by raviikushh** [comment]
Okay I'll give you update soon.

**Comment by VivekChatterjee** [comment]
Sure

**Comment by raviikushh** [comment]
Hello , here's a issue I tried many possible solutions to install Reactjs in already existing project but lots of problems are occuring.
If you give me permission to make a new React Project and then I'll make changes in that project that will be similar to yours and then I'll make a PR and You accept it/ merge it.

**Comment by raviikushh** [comment]
- It will be best if I create a new React Project and make all the changes according to your portfolio website

**Comment by VivekChatterjee** [comment]
It will be a lot of effort @raviikushh 
If it is okay with you, you can go ahead but all the functionalities should be the same.

**Comment by raviikushh** [comment]
> It will be a lot of effort @raviikushh If it is okay with you, you can go ahead but all the functionalities should be the same.

Yeah but this is the best option for now. I'll make changes according to your project. I think I can complete it by tomorrow.

**Comment by VivekChatterjee** [comment]
@raviikushh no rush, you can take your time.

**Comment by raviikushh** [comment]
> @raviikushh no rush, you can take your time.

Sure

**Comment by raviikushh** [comment]
I made a PR please check and review it.
